src/main/java/uk/gov/companieshouse/pscstatementdataapi/services/PscStatementService.java

```diff
@@ -112,19 +115,26 @@ private void saveToDb(String contextId, String companyNumber, String statementId
 
   private Created getCreatedFromCurrentRecord(String companyNumber,String statementId) {
     Optional<PscStatementDocument> document = pscStatementRepository.getPscStatementByCompanyNumberAndStatementId(companyNumber, statementId);
-    return document.isPresent() ? document.get().getCreated(): null;
+    return document.map(PscStatementDocument::getCreated).orElse(null);
```

Comment:
Happy for this to stay in but maybe add a note for testing since it'll impact the put request

Reply:
I'll take it out for now, figured it was cleaner but not worth testing again for super minor refactoring